app/models/permission.py

```diff
@@ -1,13 +1,17 @@
 from app.models import db
 
+from app.models.base import SoftDeletionModel
 
-class Permission(db.Model):
+
+class Permission(SoftDeletionModel):
```

Comment:
Here as well. Don't add more soft deletion. It has already complicated matters a lot

Reply:
:+1: